frontend/src/pages/admin/project-list.js

```diff
@@ -44,216 +60,177 @@ function projectToRow (project, onDeleteClick) {
-              />
-            </CardBody>
-          </Card>
-        </PageSection>
-        <Modal
-          title="Confirm Delete"
-          variant="small"
-          isOpen={this.state.isDeleteModalOpen}
-          onClose={this.onDeleteModalClose}
-          actions={[
-            <Button
-              key="delete"
-              variant="danger"
-              ouiaId="admin-projects-modal-delete"
-              isLoading={this.state.isDeleting}
-              isDisabled={this.state.isDeleting}
-              onClick={this.onModalDeleteClick}
-            >
-              {this.state.isDeleting ? 'Deleting...' : 'Delete'}
-            </Button>,
-            <Button
-              key="cancel"
-              variant="secondary"
-              ouiaId="admin-projects-modal-cancel"
-              isDisabled={this.state.isDeleting}
-              onClick={this.onDeleteModalClose}
-            >
-              Cancel
-            </Button>
-          ]}
-        >
-          Are you sure you want to delete &ldquo;{this.state.selectedProject && this.state.selectedProject.title}&rdquo;? This cannot be undone!
-        </Modal>
-      </React.Fragment>
-    );
-  }
-}
+          <Flex>
+            <FlexItem>
+              <Button
+                aria-label="Add project"
+                variant="secondary"
+                title="Add project"
+                ouiaId="admin-projects-add"
+                component={(props) => <Link {...props} to="/admin/projects/new" />}
+              >
+                <PlusCircleIcon /> Add Project
+              </Button>
+            </FlexItem>
+          </Flex>
+        </Flex>
+      </PageSection>
+      <PageSection className="pf-u-pb-0">
+        {!isEmpty &&
+        <Card>
+          <CardBody className="pf-u-p-0">
+            <FilterTable
+              columns={COLUMNS}
+              rows={filteredProjects ? filteredProjects.map((p) => projectToRow(p)) : [getSpinnerRow(4)]}
+              activeFilters={activeFilters}
+              filters={[
+                <TextInput
+                  type="text"
+                  id="filter"
+                  placeholder="Search for project..."
+                  value={filterText}
+                  onChange={onFilterChange}
+                  style={{height: 'inherit'}}
+                  key="filterText"
+                />
+              ]}
+              pagination={{
+                pageSize: pageSize,
+                page: page,
+                totalItems: totalItems
+              }}
+              isEmpty={isEmpty}
+              isError={isError}
+              onSetPage={(_event, value) => {setPage(value);}}
+              onSetPageSize={(_event, value) => {setPageSize(value);}}
+            />
+          </CardBody>
+        </Card>
+        }
+        {isEmpty && !filterText && <EmptyObject headingText='No Projects found' bodyText='Create your first project' returnLink='/admin/projects/new' returnLinkText='Add Project'/>}
```

Comment:
It's enough here to check if `projects.length === 0` - it means backend has returned 0 projects and we don't care if filter has been set or not since there are no projects at all